NEWS.rst

```diff
@@ -2,6 +2,18 @@
 Release Notes
 =============
 
+----------------------------
+pkgcore 0.12.16 (2022-10-04)
+----------------------------
+
+- portage_conf: fix timeout settings for rsync (#368, Arthur Zamarin)
+
+- fetch.custom: fix commands with `\`` variable escaping (Arthur Zamarin)
```

Comment:
wait, sorry, is double ` like this intended?

Reply:
No, thanks for catching it